### ISSUE TITLE
"Firefox Can't Open This Page"

### ISSUE THREAD

**Issue by pesader** [migueravila/Bento#66]
I've setup Bento as my New Tab page, using the "Custom New Tab Page" extension, but I was getting this error from firefox after setting  `  openInNewTab: false` in config.js:
![image](https://user-images.githubusercontent.com/65264536/155332837-bf82a1e8-8f5b-4b1f-8c3a-cdccc2bf7f2d.png)

After a bit of fiddling, it seems like this feature must be enabled in the "Custom New Tab Page" extension preferences:

![image](https://user-images.githubusercontent.com/65264536/155334623-10927527-877d-497c-b08e-7b17811ab8e9.png)

And indeed it fixes the issue!
May I submit a PR documenting this under "Usage" > ":heavy_plus_sign: As New Tab"?

**Comment by migueravila** [comment]
Hi @pesader totally! Make a PR!

**Comment by pesader** [comment]
Done :)
Please, let me know if you want me to include the screenshot as well.

**Comment by migueravila** [comment]
Awesome @pesader! This issue can be closed now!

**Comment by adriankarlen** [comment]
I know this was closed as solved, however when i faced this issue ticking this setting did not change the outcome. This may very well be due to other extensions I have installed. However by adding "_top" as target instead of empty string in buttons.js and list.js. Then this setting don't need to be set at all. (This is also the recommended approach suggested by the addon-author [here](https://github.com/MethodGrab/firefox-custom-new-tab-page/issues/1)).

I feel like this would be more robust solution for the issue than relying on another extension.
